openslides/core/static/js/core/base.js

```diff
@@ -48,10 +48,12 @@ angular.module('OpenSlidesApp.core', [
     'REALM',
     'ProjectorID',
     '$q',
+    '$timeout',
+    'operator',
     'ErrorMessage',
-    function (DS, REALM, ProjectorID, $q, ErrorMessage) {
+    function (DS, REALM, ProjectorID, $q, $timeout, operator, ErrorMessage) {
         var socket = null;
-        var recInterval = null;
+        var retryConnectCallbacks = [];
```

Comment:
Do we want to support multiple retry callbacks? I do not see any usecase for this.